src/components/GroupDetail/GroupRole.js

```diff
@@ -1,112 +1,110 @@
-import React, { Fragment } from 'react';
+import React, { Fragment } from "react";
 import {
-    Grid,
-    Card,
-    Button,
-    TextareaAutosize,
-    Typography,
-} from '@material-ui/core';
-import LockIcon from '@material-ui/icons/Lock';
-import { Skeleton } from '@material-ui/lab';
-import PropTypes from 'prop-types';
+  Grid,
+  Card,
+  Button,
+  TextareaAutosize,
+  Typography,
+} from "@material-ui/core";
+import "./default.css";
+import LockIcon from "@material-ui/icons/Lock";
+import { Skeleton } from "@material-ui/lab";
+import PropTypes from "prop-types";
```

Comment:
1. Please dont include changes from files other than need for this PR, 
1. even if it is just a styling change.
1. This might cause merge conflicts in the future if other persons are working on the same file in their PR.